src/webui/www/private/views/preferences.html

```diff
@@ -1760,9 +1760,13 @@
                     alert("Could not contact qBittorrent");
                 },
                 onSuccess: function(ifaces) {
-                    if (!ifaces)
+                    if (!ifaces || !Array.isArray(ifaces))
                         return;
 
+                    // add the current network interface to the options if needed
+                    if (!ifaces.find((item) => item.value === default_iface))
```

Comment:
I would use `some()` since it will return a boolean which suits better for checking existence.
```suggestion
                    if (!ifaces.some((item) => item.value === default_iface))
```